creator-node/src/diskManager.ts

```diff
@@ -110,10 +97,33 @@ export async function computeFilePath(cid: string, ensurePathExists = true) {
   // in order to easily dev against the older and newer paths, the line below is the legacy storage path
   // const parentDirPath = getConfigStoragePath()
 
+  return parentDirPath
+}
+
+/**
+ * Construct the path to a file or directory given a CID
```

Comment:
i know you didn't write the original comments here but can we add a comment about when you would use computeFilePath vs computeFilePathAndEnsureIfExists? seems like you use computeFilePath in read only (/ipfs route) or latency sensitive operations like sync

Reply:
changed 👍